test/system/500-networking.bats

```diff
@@ -82,6 +82,9 @@ load helpers
 
 # "network create" now works rootless, with the help of a special container
 @test "podman network create" {
+    if ! is_rootless; then
+        skip_if_remote "FIXME: Test flakes very frequently for yet-unknown reason(s)"
```

Comment:
To make this easier for the 7808-fixer-person to find, please say something like "FIXME: pending #7808"

Reply:
oh there's an open issue? Excellent, yes I'll pop that in there.